app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -253,6 +253,16 @@ def textual_aggregate_disk_capacity
     {:label => "Total Disk Capacity", :value => number_to_human_size(@record.aggregate_disk_capacity.gigabytes, :precision => 2)}
   end
 
+  def textual_cinder_disk_usage
+    return nil unless @record.respond_to?(:block_storage?) && @record.block_storage?
+    {:label => "Cinder Disk Usage", :value => number_to_human_size(@record.cloud_cinder_disk_usage.bytes, :precision => 2)}
+  end
+
+  def textual_swift_disk_usage
+    return nil unless @record.respond_to?(:object_storage?) && @record.object_storage?
+    {:label => "Swift Disk Usage", :value => number_to_human_size(@record.cloud_swift_disk_usage.bytes, :precision => 2)}
+  end
```

Comment:
UI lookup should take it from the indexes in textual_group_host_totals, so naming that and the methods correctly should do it.

Label will be then created from e.g. cinder_disk_usage, there is a bug with these indexes not being translated now, so it just calls titleize (but that will be just solved for you :-))

So for you is enough to call the index block_storage_disk_usage which creates label Block Storage Disk Usage